src/test-app.js

```diff
@@ -84,18 +87,31 @@ async function updateFilesWithinTestApp(info) {
   );
 }
 
+/**
+ * @param {string} pkgName
+ */
+function packagePath(pkgName) {
```

Comment:
`fse` has `readJSON`, can we use that instead? (just for consistency and having less code to maintain)